examples/examples-release-11/src/main/java/io/kubernetes/client/examples/ProtoExample.java

```diff
@@ -53,8 +53,7 @@ public static void main(String[] args) throws IOException, ApiException, Interru
     System.out.println(ns);
     if (ns.object != null) {
       namespace =
-          ns.object
```

Comment:
There is a delta between formatting w/ Java8 and formatting w/ Java11+ *sigh* 

You'll need to revert this.

Reply:
Wait, so if I run with Java8 it will work? This format change was created by the spotless plugin AFAIK.

Reply:
Looks like it :-)